Automattic/vip
# feat(dev-env): Adds automigration logic to dev-env

## Description

This adds a versioning to local environment. That way we can trigger the rebuild of `instance_data.json` as well as `lando.yml` files.
This is useful if we add new feature that touches one of these or both files. E.g. Autologin feature

## Steps to Test


Start environment and watch for messages about version:
```
Running validation steps...
✓ Check for Docker installation 
✓ Check for docker-compose installation 
✓ Check Docker connectivity 
✓ Check DNS resolution 
The most recent WordPress version available is: 6.2
Environment WordPress version is: 6.2  ... 😎 nice! 
Current local environment version is: 1.2.0
Environment version updated to: 2.0.0
```
